src/transformers/integrations.py

```diff
@@ -1064,8 +1087,9 @@ def get_run(cls, trainer):
 
     def on_log(self, args, state, control, logs: Optional[Dict[str, float]] = None, **kwargs):
         if logs:
-            for k, v in logs.items():
-                self.run[k].log(v)
+            for name, value in rewrite_logs(logs).items():
+                if isinstance(value, (int, float)):
```

Comment:
Don't we want to write some warning that we deliberately skipped some items?

How other integrations solve that problem?

Reply:
They even don't check it (only a single one without warnings). It's for future purposes as we're going to extend types that are supported by HuggingFace internally.